### PR TITLE
Improve recipe and add Ogre 13.5.0

### PR DESCRIPTION
**Pull request by joakimono**
This work also fixes Issue #8 for Ogre 13.5.0

### CODE REVIEW

File: .github/workflows/ci-conan-clang.yml [hunk 1]
```diff
@@ -19,17 +20,16 @@ jobs:
...
       fail-fast: false
       matrix:
-        version: [13.4.0]
+        version: [13.4.0, 13.5.0]
         compiler_version: [9, 10, 11]
         build_type: [Debug, Release]
         os: [ubuntu-20.04]
-        configs: ["True", "False"]
-        enable_double:  ["True", "False"]
+        configs: ['True']  # We don't build FhSim with clang
```
Comment (karljohanreite) [inline .github/workflows/ci-conan-clang.yml:27]: Kunne hatt et bedre navn enn configs? Eksempelvis "use_double"?

File: .github/workflows/ci-conan-msvc.yml [hunk 2]
```diff
@@ -43,11 +43,11 @@ jobs:
...
-            -o ogre3d:rendersystem_vulkan=${{ matrix.configs }} `
-            -o ogre3d:plugin_glslang=${{ matrix.configs }} `
-            -o ogre3d:component_bullet=${{ matrix.configs }} `
-            -o ogre3d:enable_double=${{ matrix.enable_double }} `
-            -b missing -b outdated . ${{ matrix.version }}@sintef/stable
+            -o ogre3d:with_sdl=${{ matrix.configs == 'fhsim' && 'False' || 'True' }} `
+            -o ogre3d:rendersystem_vulkan=${{ matrix.configs == 'fhsim' && 'False' || 'True' }} `
+            -o ogre3d:plugin_glslang=${{ matrix.configs == 'fhsim' && 'False' || 'True' }} `
+            -o ogre3d:enable_double=${{ ( matrix.configs == 'fhsim' && matrix.version != '13.4.0' ) && 'True' || 'False' }} `
+            -b missing -b outdated . ${{ matrix.version }}@sintef/${env:SINTEF_CHANNEL}
```
Comment (karljohanreite) [inline .github/workflows/ci-conan-msvc.yml:50]: Usikker på hvorfor du har "env:" her og ikke i gcc-versjon.

**Review by karljohanreite — APPROVED** [review]
Looks good. Please consider comments.

File: .github/workflows/ci-conan-clang.yml (hunk 1 above)
Comment (joakimono) [reply .github/workflows/ci-conan-clang.yml:27]: For clang we don't build with `enable_double=True`. We could have skipped the matrix item, but maybe in the future we will add meaningful configurations, like in the case msvc and gcc

File: .github/workflows/ci-conan-msvc.yml (hunk 2 above)
Comment (joakimono) [reply .github/workflows/ci-conan-msvc.yml:50]: This is windows-specific syntax

**Pull request merged**